hack/verify-spelling.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Using bash now because a) it's part of the new big verify job which uses bash anyway and b) because the hack/lib.sh requires bas hnow due to the while-read loop redirect.